mycore.org/content/de/documentation/basics/mcrobject/mcrobj_datatypes.html

```diff
@@ -292,6 +293,37 @@ <h2>Freier Text (MCRMetaLangText)</h2>
 </titles>{{< /highlight >}}
     </div>
 
+    <div>
+      <h2>Freier Text mit Datumsangabe (MCRMetaDateLangText)</h2>
+      <p>
+        Der Basistyp MCRMetaDateLangText dient der Speicherung einer Liste von Textabschnitten mit zugehöriger Sprachangabe und einer Datumsangabe.
+        Die Datumsangabe kann optional im <code>date</code> Attribut erfasst werden. Für den wert dieses Attributs gelten die selben regeln wie
+       für den Wert von MCRMetaISO8601Date. Über das <code>form</code> Attribut kann noch spezifiziert werden, in welcher Form der Text geschrieben ist. 
+      </p>
+      <p>
+        Optional kann ein <code>format</code>-Attribut verwendet werden. Dies erzwingt für das Datum das angegebene Format. So ist bei
+        der Formatangabe <code>YYYY</code> das Datum 2006-01 ungültig. Ohne die Formatangabe hingegen ist das gleiche Datum gültig,
+        weil es dem unterstützen Format <code>YYYY-MM</code> entspricht.
+      </p>
+      <p>XML-Syntax des Metadaten-Basistyps MCRMetaDateLangText:</p>
+      {{< highlight xml "linenos=table">}}
+<tag class="MCRMetaDateLangText" heritable="...">
+  <subtag type="..." xml:lang="..." form="..." date="..." format="...">
+    ...
+  </subtab>
+  ...
+</tag>{{< /highlight >}}
+
+      <p>Beispiel des Metadaten-Basistyps MCRMetaDateLangText:</p>
+
+      {{< highlight xml "linenos=table">}}
+<titles class="MCRMetaDateLangText" heritable="true">
```

Comment:
<title> sounds unusual for this scenario, maybe something like messages /  message  would be a better example element

Reply:
???

Reply:
I asked you, to change the name of the element title